modules/imgproc/src/smooth.simd.hpp

```diff
@@ -1376,6 +1381,12 @@ void vlineSmooth3N121<uint8_t, ufixedpoint16>(const ufixedpoint16* const * src,
     for (; i < len; i++)
         dst[i] = (((uint32_t)(((uint16_t*)(src[0]))[i]) + (uint32_t)(((uint16_t*)(src[2]))[i]) + ((uint32_t)(((uint16_t*)(src[1]))[i]) << 1)) + (1 << 9)) >> 10;
 }
+template <>
+void vlineSmooth3N121<uint16_t, ufixedpoint32>(const ufixedpoint32* const * src, const ufixedpoint32*, int, uint16_t* dst, int len)
```

Comment:
Does it make sense to specialize the template without vectorization? I think generic version should work well. Is there perf. difference?

Reply:
When FT is fixedpoint**, my compiler can't process the generic template correctly.
Anyway I will add SIMD optimization into this.

Reply:
Could you please update `hlineSmooth3N121` as well?